roles/openshift_facts/library/openshift_facts.py

```diff
@@ -1568,7 +1620,7 @@ def get_defaults(self, roles, deployment_type):
                                       etcd_hosts='', etcd_port='4001',
                                       portal_net='172.30.0.0/16',
                                       embedded_etcd=True, embedded_kube=True,
-                                      embedded_dns=True, dns_port='53',
+                                      embedded_dns=True, dns_port='8053',
```

Comment:
dns_port should probably be set in `set_dnsmasq_facts_if_unset` setting 8053 only if `use_dnsmasq` is `True`.